src/components/MAPI/apps/AppList/utils.tsx

```diff
@@ -249,3 +250,13 @@ export function mapAppCatalogEntriesToAppPageApps(
     };
   });
 }
+
+export function getAppCatalogEntryLogoURL(
+  appCatalogEntry: applicationv1alpha1.IAppCatalogEntry
+): string {
```

Comment:
As the fallback logic was more UI-related, I thought it would be more appropriate to move the function to the app list utilities file